mbedtls/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub mod ecp;
 pub mod hash;
 pub mod pk;
 pub mod rng;
-pub mod self_test;
+pub use mbedtls_selftest as self_test;
```

Comment:
Actually this is apparently a public API change (see change in mbedtls_self_tests.rs). So we still need to have a mod self_test that keeps the API the same.